rpc/lib/server/http_server.go

```diff
@@ -14,14 +14,27 @@ import (
 	"github.com/pkg/errors"
 	"golang.org/x/net/netutil"
 
-	"github.com/tendermint/tendermint/config"
 	"github.com/tendermint/tendermint/libs/log"
 	types "github.com/tendermint/tendermint/rpc/lib/types"
 )
 
-// Config is an RPC server configuration.
+// Config is a RPC server configuration.
 type Config struct {
+	// see netutil.LimitListener
 	MaxOpenConnections int
+	// mirrors http.Server#ReadTimeout
+	ReadTimeout time.Duration
+	// mirrors http.Server#WriteTimeout
+	WriteTimeout time.Duration
+}
+
+// DefaultConfig returns a default configuration.
+func DefaultConfig() *Config {
+	return &Config{
+		MaxOpenConnections: 0, // unlimited
+		ReadTimeout:        3 * time.Second,
+		WriteTimeout:       3 * time.Second,
```

Comment:
lowered from 20 to 3 seconds. Why it was 20 seconds in the first place?

Reply:
Makes sense 👍  But make sure the default of WriteTimeout > TimeoutBroadcastTxCommit (which is 10 s; see: https://github.com/tendermint/tendermint/pull/3443#discussion_r266830038).

Reply:
Set to slightly modified Nginx defaults.